manifest.json

```diff
@@ -1,28 +1,46 @@
 {
-  "name": "@deskpro-apps/my-app",
-  "title": "My App",
-  "description": "My Deskpro app",
-  "version": "0.0.4",
+  "name": "@deskpro-apps/wrike",
+  "title": "Wrike",
+  "description": "Wrike integration for Deskpro",
+  "version": "0.0.0",
   "scope": "agent",
   "isSingleInstall": false,
   "hasDevMode": true,
-  "targets": [{ "target": "ticket_sidebar", "entrypoint": "index.html" }],
+  "targets": [
+    {
+      "target": "ticket_sidebar",
+      "entrypoint": "index.html"
+    }
+  ],
   "settings": {
-    "example": {
-      "title": "Example Setting",
+    "access_token": {
+      "title": "Access Token",
       "description": "Example backend only setting",
       "type": "string",
-      "isRequired": false,
+      "isRequired": true,
       "isBackendOnly": true
     }
   },
+  "entityAssociations": {
+    "linkedTasks": {
```

Comment:
Change to `linkedWrikeTasks` as this will compete with other installed apps when mounting custom fields and we need to differentiate them